### DISCUSSION TITLE
Return Url check case sensitive

### DISCUSSION THREAD

**Discussion by agkoutis** [simpleidserver/SimpleIdServer#621] [Q&A]
Hello,

It seems that the validator of the authorization request is case-sensitive against the db client-allowed return URL.

For the IIS virtual apps there is a problem because if a user types the url of the Client App  in the browser, identity model will auto-populate the return url in the authorized request using this url. 

Example

SID handles https://iis/sid.admin deferred from the https://iis/sid.Admin

"error": "invalid_request",
"error_description": "redirect_uri https://iis/sid.Admin/signin-oidc is not correct",

Is there any workaround?

**Comment by simpleidserver** [comment]
Hello,

Currently, SimpleIdServer is using regular expressions to check the "Redirection URL." 
You can set the redirection URL to something like this: `https://iis/(s|S)(i|I)(d|D).(a|A)(d|D)(m|M)(i|I)(n|N)`.

In the meantime, I have created a new ticket, #622, to support case-sensitive redirection URLs.

Kind regards,

SID

**Comment by agkoutis** [comment]
Great news!

**Comment by simpleidserver** [comment]
The support of case sensitive redirect_url is available in the branch `release/4.0.5`.
